app/graphql/types/input/new_organization_type.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+Types::Input::NewOrganizationType = GraphQL::InputObjectType.define do
+  name 'NewOrganization'
+  description <<~DESCRIPTION
+    Contains all fields that are possible to set when creating a new
```

Comment:
It was introduced with Ruby 2.3 and we have 2.4 as the target version in the rubocop config... Looks like Hound runs on 2.2 or less. In any case, let's try to update the rubocop config to 2.4. I've seen a lot of weird behaviour - maybe this changes something...